src/nycdb/sql/oca/first_last.sql

```diff
@@ -7,6 +7,7 @@
 -- FROM: https://wiki.postgresql.org/wiki/First/last_(aggregate)
 
 -- Create a function that always returns the first non-NULL item
+DROP FUNCTION IF EXISTS public.first_agg(anyelement, anyelement) CASCADE;
```

Comment:
Doesn't `CREATE OR REPLACE FUNCTION` do the same thing as dropping the function before re-creating it? or was there some other problem that required the addition of this line?

Reply:
it usually does, but I remember having some issue where this seemed better. I was a while ago, but believe it had to do with having a function with the same name but different argument types and `create or replace` would raise an error rather than just replacing the function in those cases.